README.md

```diff
@@ -1 +1,30 @@
 
+# Run and Shoot application
+
+# Carleton University
+    Authors
+        -Mahsa Layeghi
+        -Parichehreh Firoozishahmirzadi
+        -Jin Xu
+        -Jaser El-Habrouk
+
+The application is for fun purposes where the player has to reach a certain goal on a map while trying to avoid barriers and arrowns.
+
+# Folder Organization
+/src folder contains all the functions needed for the game as well as the main function.
+/test folder contains all the test functions for the game functions.
+/include has all the include files needed for the application's functions
+
+# Compilation
```

Comment:
The reader need to compile the whole program completely using the information provided in readme. This does not have this information.
Please check Makefile to see how to compile and run the program.
Also, you need to mention the program dependencies(SDL library) and a link to wiki.